scripts/rsync.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04
+
+ARG DOMINO_UID=12574
```

Comment:
These must be set dynamically based on what is specified in the CRD, right?

Reply:
The other way around, I suppose... Here, the IDs are "baked into" the image, so DCO controller must use the same when launching the sidecar. Those values are hardcoded as constants in `pkg/cluster/mpi/mpi.go`. So, there are two places in the code containing hardcoded sets of values that must be consistent with each other. Which is OK.

Reply:
I think this is wrong, because Nucleus still needs to know how to SSH into the `rsync` container. I supposed it can rely on the hardcoding, but that seems brittle.

Reply:
Yeah, I see your point. We'd have to create the user in the runtime, similar to the main container, instead of baking it in. But if we must be able to specify the user account via CRD, that's OK. I've just though that my approach will be more elegant (through less flexible).

Reply:
I guess in the interest of avoiding running as root, lets keep this hardcoded in both Nucleus and here (i.e. what you have now).